### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @MolinaElGringo

### DISCUSSION THREAD

**Discussion by MolinaElGringo** [heig-vd-dai-course/heig-vd-dai-course#47] [Show and tell]
[Link to my Github Repo](https://github.com/MolinaElGringo/FirstProjectMaven)

**Comment by ludelafo** [comment]
Thank you!

The README file should be named `README.md`. In your repository, it is named `READ.md`.

No problem for me if the README is in French, but please be aware that no everyone will understand your projects if you publish your work on GitHub if you do so.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
